samples/maps/demo/geoheatmap/demo.details

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This demo is beautiful and tells a friendly story, but unfortunately I think it's too slow to use in the demo pages. It is apparent when drag-rotating the globe. As I see it, this would be a great demonstration when we get the `interpolation` feature implemented, which will presumably fix the performance issue because it uses canvas rather than SVG.

My suggestion is to move this demo to `studies` while waiting for the interpolation feature, then use it more or less as is to demonstrate that issue.

@KacperMadej any thoughts?